# HG changeset patch
# User Rik <rik@octave.org>
# Date 1381962280 25200
#      Wed Oct 16 15:24:40 2013 -0700
# Node ID ea9df126c9a52fd5350eff01d412cab143ac8a48
# Parent  b7670c05731b3eb32fd3628b27e5962b3478c931
quiver3.m: Turn plot box off, and grid on, for Matlab compatibility.

* scripts/plot/draw/quiver3.m: Turn plot box off, and grid on, for Matlab
compatibility.

diff --git a/scripts/plot/draw/quiver3.m b/scripts/plot/draw/quiver3.m
--- a/scripts/plot/draw/quiver3.m
+++ b/scripts/plot/draw/quiver3.m
@@ -75,17 +75,18 @@ function retval = quiver3 (varargin)
   if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
     unwind_protect
       hax = newplot (hax);
       htmp = __quiver__ (hax, true, varargin{:});
 
       if (! ishold (hax))
-        set (hax, "view", [-37.5, 30]);  # 3D view
+        set (hax, "view", [-37.5, 30], "box", "off",
+                  "xgrid", "on", "ygrid", "on", "zgrid", "on");
       endif
     unwind_protect_cleanup
       if (! isempty (oldfig))
         set (0, "currentfigure", oldfig);
       endif
     end_unwind_protect
   endif
 
